lib/jsdom/living/file-reader.js

```diff
@@ -69,7 +69,7 @@ module.exports = function createFileReader(window) {
     if (self.readyState === self.LOADING) {
       throw new DOMException(DOMException.INVALID_STATE_ERR);
     }
-    if (file[blobSymbols.closed]) {
+    if (file.isClosed) {
```

Comment:
This is not correct, since authors can override `isClosed` using `Object.defineProperty`.